src/views/AddProductModal.vue

```diff
@@ -117,7 +117,6 @@ export default defineComponent({
       this.store.dispatch('shipment/addShipmentItem', product)
     },
     closeModal() {
-      this.store.dispatch('product/clearSearchedProducts')
```

Comment:
Unwanted change.

Reply:
Removed this as we are dispatching the same action in onDidDismiss().